src/main/webapp/WEB-INF/views/cars/add.jsp

```diff
@@ -0,0 +1,19 @@
+<%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core" %>
+<%@ page contentType="text/html;charset=UTF-8" language="java" %>
+<html>
+<head>
+    <title>Add car</title>
+</head>
+<body>
+<h1>Please, fill car form</h1>
+    <form method="post" action="${pageContext.request.contextPath}/cars/add">
+    Model <input type="text" name="model"><br/>
+    Manufacturer <select name="manufacturer_id">
+        <c:forEach items="${manufacturers}" var="manufacturer">
+            <option value="${manufacturer.getId()}"> ${manufacturer.getName()} </option>
```

Comment:
we can write just `manufacturer.id` and so on
just for you to know :)

Reply:
Thanks, I know. It isn’t necessary.